test/terraform/module-pinning.bats

```diff
@@ -1,6 +1,10 @@
 load 'lib'
 
 function setup() {
+  if ! which terraform-config-inspect; then
```

Comment:
Note we use the other pattern for more than config inspect. We should then be consistent.

Reply:
The `bats` output is a numbered list of tests with `ok` or `not ok`. Those should be tests of the code, not tests of the test harness. This pattern of testing in the setup will still cause the test to fail with an error message, but does not add to the number of tests or give an `ok`/`not ok` on the code being tested. If we are using `@test` elsewhere to test the test harness, then yes, we should get rid of those, too.